<launch>
  <arg name="twist_cmd_topic" default="/cmd_vel"/>
  <arg name="joy_topic" default="/joy"/>
  <arg name="joy_number" default="0"/>
  <arg name="deadzone" default="0"/>

  <param name="joy_node/dev" value="/dev/input/js$(arg joy_number)"/>
  
  <node name="joy" pkg="joy" type="joy_node">
    <param name="deadzone" value="$(arg deadzone)"/>
  </node>

  <node name="joy_teleop" pkg="alberto_bringup" type="alberto_joy_teleop.py" output="screen">
    <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)"/>
    <param name="joy_topic" value="$(arg joy_topic)"/>
  </node>

</launch>